src/common/_fonts.scss

```diff
@@ -39,11 +39,12 @@
-    format("truetype");
+  src: url("../fonts/acta-display-extra-bold-no-tittle.woff2") format("woff2"),
```

Comment:
I did this to fix font jumping on the animation. If people don't have your fonts loaded beforehand the whole thing looks disgusting. Grappled with Baila about this and this was the solution that worked best.

Reply:
Hmmm, unicode-range though... I still think it would be best to inline this somehow. I can give you the tool I used

Reply:
I wasn't aware the inlining was to avoid a FOUT. Still the base64 representation of this font causes for an awfully long line.

Made some changes then:
* Added a PostCSS plugin to inline fonts based on path, allowing us to keep `base.scss` cleaner;
* Restored a smaller version of _Acta Display_ to be used in all titles with the most common English characters. The unicode range along with the `subset` word in the name of the font family should make that painfully clear;
* The regular font will be inlined to be loaded into the HTML. The tittleless version will be loaded normally since the splash should give us more than enough time to load it.

Reply:
The inline can be a bit big. The base 64 only had the characters on the title.

Reply:
Anyway, seems ok

Reply:
> The inline can be a bit big. The base 64 only had the characters on the title.

True, but:
- That inline did not replace _Acta Display_ completely, just in the homepage. This one replaces all ocurrences, including in blog posts where the content is dynamic, that's why I included more characters.
- The net result of removing _Acta Display_ and adding this one, without considering the inline version removed, is around -20KB.

Reply:
Good